### PR TITLE
ignition-msgs6, ignition-fuel-tools5: Dome dependencies

### PR DESCRIPTION
**Pull request by chapulina**
For this PR: https://bitbucket.org/ignitionrobotics/ign-fuel-tools/pull-requests/127/

### CODE REVIEW

File: Formula/ignition-fuel-tools5.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+class IgnitionFuelTools5 < Formula
+  desc "Tools for using Fuel API to download robot models"
+  homepage "https://ignitionrobotics.org"
+  url "https://bitbucket.org/ignitionrobotics/ign-fuel-tools/get/5b7d9e7be787619a3b125b0675912e1a21403fc6.tar.bz2"
+  sha256 "8f8a25b56265dd3853383100f9b73b9af22fdb06980c8d917dfb0316afb16229"
+  revision 1
```
Comment (scpeters) [inline Formula/ignition-fuel-tools5.rb:6]: new formulae shouldn't have a revision or bottle block

and they should have an explicit version

**Comment by scpeters** [comment]
testing with this PR:

* [Build Status](https://build.osrfoundation.org/job/ignition_fuel-tools-ci-pr_any-homebrew-amd64/280/) https://build.osrfoundation.org/job/ignition_fuel-tools-ci-pr_any-homebrew-amd64/280/

**Comment by scpeters** [comment]
we can merge that without building bottles if that passes

**Comment by chapulina** [comment]
All the checks have passed now, I don't remember if they had passed before. I triggered a new build: [Build Status](https://build.osrfoundation.org/job/ignition_fuel-tools-ci-pr_any-homebrew-amd64/282/)

**Comment by scpeters** [comment]
ok, let's do it

**Pull request merged**